<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
<title>Xihong Lin's Group @ Harvard School of Public Health</title>
<meta name="keywords" content="Xihong Lin, Harvard University, Harvard School of Public Health, HSPH, Biostatistics">
<meta name="description" content="Professor Xihong Lin's group at Department of Biostatistics, Harvard School of Public Health">
<link rel="stylesheet" type="text/css" href="css/mystyle.css">
</head>

<body>

<div class="page">

<div class="banner" id="top"></div>

<script language="JavaScript">
<!--
function showyear(i,yearmax) {
  for(var iter=2001; iter<=yearmax; iter++) {
    if(iter==i) {
        var myid=document.getElementById('y'+iter);
        myid.style.display="";
    } else {
        var myid=document.getElementById('y'+iter);
        myid.style.display="none";
    }
  }
}
-->
</script>

<div class="navpart"> 
	<ul class="navbar">
	<li><a href="index.html" class="nav">Home</a></li>
	<li><a href="research.html" class="nav">Research</a></li>
	<li><a href="people.html" class="nav">People</a></li>
	<li><a href="publications.html" class="currentnav">Publications</a></li>
	<li><a href="software.html" class="nav">Software</a></li>
	<li><a href="data.html" class="nav">Data</a></li>
	<li><a href="links.html" class="nav">Links</a></li>
	</ul>
</div>

<div class="listbar">
	<ul class="listbox">
	<li class="group">Publications</li>
<li><a href="javascript:showyear(2023,2023)" class="jumpto">2023</a></li>
	<li><a href="javascript:showyear(2022,2023)" class="jumpto">2022</a></li>
	<li><a href="javascript:showyear(2021,2023)" class="jumpto">2021</a></li>
	<li><a href="javascript:showyear(2020,2023)" class="jumpto">2020</a></li>
	<li><a href="javascript:showyear(2019,2023)" class="jumpto">2019</a></li>
	<li><a href="javascript:showyear(2018,2023)" class="jumpto">2018</a></li>
	<li><a href="javascript:showyear(2017,2023)" class="jumpto">2017</a></li>
	<li><a href="javascript:showyear(2016,2023)" class="jumpto">2016</a></li>
	<li><a href="javascript:showyear(2015,2023)" class="jumpto">2015</a></li>
	<li><a href="javascript:showyear(2014,2023)" class="jumpto">2014</a></li>
	<li><a href="javascript:showyear(2013,2023)" class="jumpto">2013</a></li>
        <li><a href="javascript:showyear(2012,2023)" class="jumpto">2012</a></li>
	<li><a href="javascript:showyear(2011,2023)" class="jumpto">2011</a></li>
	<li><a href="javascript:showyear(2010,2023)" class="jumpto">2010</a></li>
        <li><a href="javascript:showyear(2009,2023)" class="jumpto">2009</a></li>
        <li><a href="javascript:showyear(2008,2023)" class="jumpto">2008</a></li>
        <li><a href="javascript:showyear(2007,2023)" class="jumpto">2007</a></li>
        <li><a href="javascript:showyear(2006,2023)" class="jumpto">2006</a></li>
        <li><a href="javascript:showyear(2005,2023)" class="jumpto">2005</a></li>
        <li><a href="javascript:showyear(2004,2023)" class="jumpto">2004</a></li>
        <li><a href="javascript:showyear(2003,2023)" class="jumpto">2003</a></li>
        <li><a href="javascript:showyear(2002,2023)" class="jumpto">2002</a></li>
        <li><a href="javascript:showyear(2001,2023)" class="jumpto">before 2002</a></li>
	</ul>
</div>


<div class="pubmain">

<div><h2>Selected Publications</h2></div>

<div id="y2023"><h3>Year 2023</h3>
<ul>

<li>Li, X., Quick, C., Zhou, H., Gaynor, S., Liu, Y., Chen, H., Selvaraj, M., Sun, R., Dey, R., Arnett D., Bielak, L., Bis, J., Blangero, J., Boerwinkle, E., Bowden, D., Brody, J., Cade, B., Correa, A., Cupples, L.A., Curran, J., De Vries, P., Duggirala, R., Freedman, B., Goring, H., Guo, X., Haessler, J., Kalyani, R., Kooperberg, C., Kral, B., Lange, L., Manichaikul, A., Martin, L., McGarvey, S., Mitchell, B., Montasser, M., Morrison, A., Naseri, T., O'Connell, J., Palmer, N., Peyser, P., Psaty, B., Raffield, L., Redline, S., Reiner, A., Reupena, M., Rice, K., Rich, S., Sitlani, C., Smith, J., Taylor, K., Vasan, R., Wilson, J., Willer, C., Yanek, L., Zhao, W., NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, TOPMed Lipids Working Group, Rotter, J., Natarajan, P., Peloso, G., Li, Z., Lin, X. (2023)  
<a href="https://www.nature.com/articles/s41588-022-01225-6"> Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies.</a><i>Nature Genetics</i> 55(1), 154-164

<li>Zhou, H., Arapoglou, T., Li, X., Li, Z., Zheng, X., Moore, J., Asok, A., Kumar, S., Blue, E.E., Buyske, S., Cox, N., Felsenfeld, A., Gerstein, M., Kenny, E., Li, B., Matise, T., Philippakis, A., Rehm, H.L., Sofia, H.J., Snyder, G., NHGRI Genome Sequencing Program Variant Functional Annotation Working Group, Weng, Z., Neale, B., Sunyaev, S.R., Lin, X. (2023). 
<a href="https://academic.oup.com/nar/article/51/D1/D1300/6814464"> FAVOR: functional annotation of variants online resource and annotator for variation across the human genome.</a><i>Nucleic Acids Research</i> 51(D1), D1300-D1311


</ul>
</div>

<div id="y2022"><h3>Year 2022</h3>
<ul>

<li>Li, Z., Li, X., Zhou, H., Gaynor, S.M., Selvaraj, M., Arapoglou, T., Quick, C., Liu, Y., Chen, H., Sun, R., Dey, R., Arnett D., Auer, P., Bielak, L., Bis, J., Blackwell, T., Blangero, J., Boerwinkle, E., Bowden, D., Brody, J., Cade, B., Conomos, M., Correa, A., Cupples, L.A., Curran, J., De Vries, P., Duggirala, R., Franceschini, N., Freedman, B., Goring, H., Guo, X., Kalyani, R., Kooperberg, C., Kral, B., Lange, L., Lin, B., Manichaikul, A., Manning, A., Martin, L., Mathias, R., Mitchell, B., Montasser, M., Morrison, A., Naseri, T., O'Connell, J., Palmer, N., Peyser, P., Psaty, B., Raffield, L., Redline, S., Reiner, A., Reupena, M., Rice, K., Rich, S., Smith, J., Taylor, K., Taub, M., Vasan, R., Week, D., Wilson, J., Yanek, L., Zhao, W., NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, TOPMed Lipids Working Group, Rotter, J., Willer, C., Natarajan, P., Peloso, G., Lin, X. (2022). <a href="https://www.nature.com/articles/s41592-022-01640-x"> A framework for detecting noncoding rare-variant associations of large-scale whole-genome sequencing studies.</a>
<i>Nature Methods</i> 19(12), 1599-1611

<li>Li, X., Yung, G., Zhou, H., Sun, R., Li, Z., Hou, K., Zhang, M.J., Liu, Y., Arapoglou, T., Wang, C., Ionita-Laza, I., Lin, X. (2022). <a href="https://pubmed.ncbi.nlm.nih.gov/35216679/"> A multi-dimensional integrative scoring framework for predicting functional variants in the human genome.</a>
<i>The American Journal of Human Genetics</i> 109(3), 446-456

<li>Li, Z., Liu, Y., Lin X. (2022). <a href="https://doi.org/10.1080/01621459.2020.1822849/"> Simultaneous detection of signal regions using quadratic scan statistics with applications in whole genome association studies.</a>
<i>Journal of American Statistical Association</i> 117(538), 823-834

<li>Liu, Y., Li, Z., Lin X. (2022). <a href="https://doi.org/10.1080/01621459.2020.1831926/"> A minimax optimal ridge-type set test for global hypothesis with applications in whole genome sequencing association studies.</a>
<i>Journal of American Statistical Association</i> 117(538), 897-908

<li>Liu, Z., Shen, J., Barfield, R., Schwartz, J., Baccarelli, A., Lin, X. (2022). <a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2021.1914634/"> Large-Scale Hypothesis Testing for Causal Mediation Effects with Applications in Genome-wide Epigenetic Studies.</a>
<i>Journal of American Statistical Association</i> 117(537), 67-81

<li>Dey, R., Zhou, W., Kiiskinen, T., Havulinna, A., Elliott, A., Karjalainen, J., Kurki, M., Qin, A., FinnGen, Lee, S., Palotie, A., Neale, B., Daly, M., Lin, X. (2022). <a href="https://www.nature.com/articles/s41467-022-32885-x/"> Efficient and accurate frailty model approach for genome-wide survival association analysis in large-scale biobanks.</a>
<i>Nature Communications</i> 13(1), 5437

<li>Gaynor, S.M., Westerman, K., Ackovic, L., Li, X., Li, Z., Manning, A., Philippakis, A., Lin, X. (2022). <a href="https://doi.org/10.1093/bioinformatics/btac272/">STAAR Workflow: A cloud-based workflow for scalable and reproducible rare variant analysis.</a>
<i>Bioinformatics</i> 38(11), 3116-3117

<li>Sun, R., Shi, A., Lin, X. (2022). <a href=" https://doi.org/10.1093/biostatistics/kxac036/"> Differences in set-based tests for sparse alternatives when testing sets of outcomes compared to sets of explanatory factors in genetic association studies.</a>
<i>Biostatistics</i> 

<li>Lin, X. (2022). <a href=" https://projecteuclid.org/journals/statistical-science/volume-37/issue-2/Lessons-Learned-from-the-COVID-19-Pandemic--A-Statisticians/10.1214/22-STS860.full/"> Lessons Learned from the COVID-19 Pandemic: A Statistician's Reflection.</a>
<i>Statistical Science</i> 37(2), 278-283

<li>Shi, A., Gaynor, S.M., Dey, R., Zhang, H., Quick, C., Lin X. (2022). <a href="https://academic.oup.com/bioinformatics/article/38/9/2661/6542303/"> COVID-19 Spread Mapper: a multi-resolution, unified framework and open-source tool.</a>
<i>Bioinformatics</i> 38(9), 2661-2663

<li>McCaw, Z.R., Gaynor, S.M., Sun, R., Lin X. (2022). <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13629/">Leveraging a surrogate outcome to improve inference on a partially missing target outcome. </a>
<i>Biometrics</i> 

<li>Li, D., Ren, H., Varelmann, D.J., Sarin, P., Xu, P., Wu, D., Li, Q., Lin, X. (2022). <a href="https://bmjopen.bmj.com/content/12/2/e053635.abstract/"> Risk assessment for acute kidney injury and death among new COVID-19 positive adult patients without chronic kidney disease: retrospective cohort study among three US hospitals.</a>
<i>BMJ open</i> 12(2), e053635

<li>Byun, J.. Han, Y., Li, Y., ...,  Lin, X.,  McKay, J., Hung, R. J., Amos, C. I, INTEGRAL-ILCCO Consortium (2022). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9373844/"> Trans-ancestry genome-wide meta-analysis of 61,047cases and 947,237 controls identifies novel genomic cross-ancestry loci contributing to lung cancer susceptibility.</a>
<i>Nature Genetics</i> 54(8), 1167-1177

<li>Hong, D., Dey, R., Lin, X.*, Cleary, B.*, Dobriban, E.* (2022). <a href="https://www.nature.com/articles/s41467-022-29389-z/"> Group testing via hypergraph factorization applied to COVID-19.</a>
<i>Nature Communication</i> 13, 1837 (* Co-corresponding authors)

<li>Selvaraj, M.S., Li, X., Li, Z., Pampana, A., Zhang, D.Y., Park, J., Aslibekyan, S., Bis, J.C., Brody, J.A., Cade, B.E. and Chuang, L.M.,..., Lin, X., Peloso, G., M., Natarajan, P. (2022) . <a href="https://www.nature.com/articles/s41467-022-33510-7/"> Whole genome sequence analysis of blood lipid levels in> 66,000 individuals.</a>
<i>Nature Communication</i> 13, 5995 



</ul>
</div>

<div id="y2021"><h3>Year 2021</h3>
<ul>

<li>Quick, C., Dey, R., & Lin, X. (2021). <a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2021.2001339"> Regression Models for Understanding COVID-19 Epidemic Dynamics With Incomplete Data.</a>
<i> Journal of the American Statistical Association</i> 116(536), 1561-1577

<li>Li, D., Gaynor, S.M., Quick, C., Chen, J., Stephenson, B., Coull, B., Lin, X. (2021). <a href="https://bmcpublichealth.biomedcentral.com/articles/10.1186/s12889-021-11060-9">Identifying US County-level characteristics associated with high COVID-19 burden.</a>
<i>BMC public health</i> 21(1), 1-10

<li>Naseri, A., Shi, J., Lin, X., Zhang, S., Zhi, D. (2021). <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009315/"> RAFFI: Accurate and fast familial relationship inference using RaPID in Large Scale Biobank Studies.</a>
<i>Plos Genetics</i> 17(1): e1009315

<li>Ke, Z., Ma, Y., and Lin, X. (2021). <a href="http://zke.fas.harvard.edu/papers/BEMA.pdf">Estimating the rank of a spiked covariance matrix by spectral quantile matching.</a>
<i>Journal of the American Statistical Association</i> 

<li>Wang, X., Ricciuti, B.,Nguyen, T.,Li,X., Rabin, M. S., Awad, M. M.,Lin, X., Johnson, B., Christiani, D. C.  (2021). <a href="https://pubmed.ncbi.nlm.nih.gov/33653773/">Association between smoking history and tumor mutation burden in advanced non-small cell lung cancer.  </a>
<i>Cancer Research</i> 81(9): 2566¨C2573

<li>Wang, X., Lin, X., Johnson, B., Christiani, D. C. (2021) .<a href="https://pubmed.ncbi.nlm.nih.gov/34115098/">Smoking history as a potential predictor of immune checkpoint inhibitor (ICI) efficacy in metastatic non-small cell lung cancer (NSCLC).</a>
<i>Journal of National Cancer Institute</i> 113(12), 1761-1769

<li>Dayan, I., Roth. H., Zhong, Z., Harouni, A., ...,  Lin, X., Wen, Y., Gilbert, F. J., Flores, M. G., Li, Q. (2021).  <a href="https://www.nature.com/articles/s41591-021-01506-3">Federated Learning used for predicting outcomes in SARS-COV-2 patients.</a>
<i>Nature Medicine</i> 27,1735-1743

<li>Wu, P., Ding, L., Li, X., Liu, S., Cheng, F., He, Q., Xiao, M., Wu, P., Hou, H., Jiang, M. and Long, P., Lin, X., Jin, X., Wang, F., Sun, C., and Wang, C. (2021) .  <a href="https://pubmed.ncbi.nlm.nih.gov/34465887/">Identifying US County-level characteristics associated with high COVID-19 burden.</a>
<i>Communications biology</i> 4(1), 1-10.




</ul>
</div>


<div id="y2020"><h3>Year 2020</h3>
<ul>

<li>Liu, Z., Barnett, I, and Lin, X. (2020)  
<a href="https://projecteuclid.org/euclid.aoas/1587002681"> Differences in principal component methods between multiple phenotype regression and multiple SNP regression in genetic association studies.</a>
<i>Annals of Applied Statistics </i> 14(1), 433-451

<li>Raffield, L. M., Lyengar, A. K., Wang, B., Gaynor, S.M., Lin, X., Li, B., Li, Y., Dupuis, J., Reiner, A., Moklke, K.R., Auer, P. TOPMed Inflammation Working Group, NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium. (2020) 
<a href="https://pubmed.ncbi.nlm.nih.gov/31883642/"> Allelic heterogeneity at the CRP locus identified by whole-genome sequencing in multi-ancestry cohorts.</a>
<i>American Journal of Human Genetics </i> 106(1), 112-120

<li>Pan, A., Liu, L., Wang, C., Guo, H., Hao, X., Wang Q., Huang, J., He, N., Yu, Ho, Lin, X.*, Wei, S.*, Wu, T.*  (2020)
<a href="https://jamanetwork.com/journals/jama/fullarticle/2764658"> Association of Public Health Interventions With the Epidemiology of the COVID-19 Outbreak in Wuhan, China.</a>
<i>Journal of the American Medical Association</i> 323(19):1915-1923 (* Co-corresponding authors)

<li>Gao, X., Coull, B., Lin, X., Vokonas, P., Hou, L., DeMeo, D.L., Litonjua, A.A., Schwartz, J.D. and Baccarelli, A.A. (2020) 
<a href="https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2768234"> Association of Neutrophil to Lymphocyte Ratio With Pulmonary Function in a 30-Year Longitudinal Study of US Veterans.</a>
<i>JAMA Network Open</i> 3(7), e2010350--e2010350

<li>Li, X., Li, Z., Zhou, H., Gaynor, S.M., Liu, Y., Chen, H., Sun, R., Dey, R., Arnett, D.K., Aslibekyan, S., Ballantyne, C.M., Bielak, L.F., Blangero, J., Boerwinkle, E., Bowden, D.W., Broome, J.G., Conomos, M.P., Correa, A., Cupples, L.A., Curran, J.E., Freedman, B.I., Guo, X., Hindy, G., Irvin, M.R., Kardia, SLR., Kathiresan, S., Khan, A.T., Kooperberg, C.L., Laurie, C.C., Liu, X.S., Mahaney, M.C., Manichaikul, A.W., Martin, L.W., Mathias, R.A., McGarvey, S.T., Mitchell, B.D., Montasser, M.E., Moore, J.E., Morrison, A.C., O'Connell, J.R., Palmer, N.D., Pampana, A., Peralta, J.M., Peyser, P.A., Psaty, B.M., Redline, S., Rice, K.M., Rich, S.S., Smith, J.A., Tiwari, H.K., Tsai, M.Y., Vasan, R.S., Wang, F., Weeks, D.E., Weng, Z., Wilson, J.G., Yanek, L.R., NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, TOPMed Lipids Working Group, Neale, B.M., Sunyaev, S.R., Abecasis, G.R., Rotter, J.I., Willer, C.J., Peloso, G.M., Natarajan, P., and Lin, X.  (2020)
<a href="https://www.nature.com/articles/s41588-020-0676-4"> 
Dynamic incorporation of multiple in-silico functional annotations empowers rare variant association analysis of large whole genome sequencing studies at scale. </a>
<i>Nature Genetics</i> 52(9), 969-983

<li>Segal, E., Zhang, F., Lin, X., King, G., Shalem, O, Shilo, S, Allen, W., Grad, Y., Greene, C., Alquaddoomi, F., Anders, S., Balicer, R., Bauman, T., Bonilla, X, Booman, G., Chan, A., Cohen, O., Coletti, S., Davidson, N., Dor, Y., Drew D., Elemento, O., Evans, G., Ewels, P., Gale, J., Gavrieli, A., Geiger, G., Hajirasouliha, I., Jerala, R., Kahles, A., Kallioniemi, O, Keshet, A., Kocarev, L., Landua, G., Meir, T., Muller, A., Nguyen, A., Oresic, M., Ovchinnikova, S., Peterson, D. , Prodanova, J., Rajagopal, J., Rtsch, G., Rossman, H., Rung, J., Sboner, A., Sigaras, A., Spector, T., Steinherz, R., Stevens, I., Vilo, J., Wilmes, P. (2020)
<a href="https://www.nature.com/articles/s41591-020-0929-x"> Building an International Consortium for Tracking Coronavirus Health Status </a>
<i>Nature Medicine</i> 26(8), 1161-1165 

<li>Sun, R. and Lin, X. (2020).   
<a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2019.1660170"> Genetic Variant Set-Based Tests Using the Generalized Berk-Jones Statistic With Application to a Genome-Wide Association Study of Breast Cancer</a>
<i>Journal of the American Statistical Association </i> 115 (531), 1079-1091

<li>McCaw, Z., Lane, J. M., Saxena, R., Redline, S., and Lin, X. (2020) .
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8643141/"> Normal Transformation Improves Genetic Association Testing with Quantitative Traits</a>
<i>Biometrics </i> 76(4), 1262-1272

<li>Allen, W. E. Altae-Tran, H., Briggs, J., Jin, X., McGee, G., Shi, A., Raghavan, R., Kamariza, M., Nova, N.,  ..., Travassos, M., Coull, B. A., Openshaw, J., Greene, C. S., Shalem, O., King. G., Probasco, R., Cheng, D.,  Silbermann, B., Zhang, F., and Lin, X. (2020).  
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8643141/"> Population-scale Longitudinal Mapping of COVID-19 Symptoms, Behavior, and Testing Identifies Contributors to Continued Disease Spread in the United States</a>
<i>Nature Human Behavior</i> 4,972¨C982

<li>Hao, X., Cheng, S., Wu, D., Wu, T.*, Lin, X.* and Wang, C.* (2020) . 
<a href="https://www.nature.com/articles/s41586-020-2554-8/"> Reconstruction of the full transmission dynamics of COVID-19 in Wuhan</a>
<i>Nature</i> 584(7821), 420-424. (* Co-corresponding authors)

<li>He, X., and Lin, X. (2020). 
<a href="https://hdsr.mitpress.mit.edu/pub/da99kl2q/release/3/"> Challenges and Opportunities in Statistics and Data Science: Ten Research Areas (with Discussions).</a>
<i>Harvard Data Science Review</i> 2(3)

<li>Lin, X. (2020). 
<a href="https://hdsr.mitpress.mit.edu/pub/yosmh9o4/release/2/"> Learning Lessons on Reproducibility and Replicability in Large Scale Genome-Wide Association Studies.</a>
<i>Harvard Data Science Review</i> 2(4)

<li>Nguyen, L.H., Drew, D.A., Graham, M.S., Joshi, A.D., Guo, C.G., Ma, W., Mehta, R.S., Warner, E.T., Sikavi, D.R., Lo, C.H. and Kwon, S., ... Lin, X.,... Zhang, F (2020). 
<a href="https://pubmed.ncbi.nlm.nih.gov/32745512/"> Risk of COVID-19 among front-line health-care workers and the general community: a prospective cohort study.</a>
<i>The Lancet Public Health</i> 5(9), e475-e483

<li>Taliun, D., Harris, D.N., Kessler, M.D., Carlson, J., Szpiech, Z.A., Torres, R., Taliun, S.A.G., Corvelo, A., Gogarten, S.M., Kang, H.M. and Pitsillides, A.N., , Lin, X.,  , ..., Abecasis, G. (2020). 
<a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2019.1660170"> Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program.</a>
<i>Nature </i> 590(7845), 290-299

<li>Sun, R., Xu. M., Li, X., Gaynor, S., Zhou, H., Bosse, Y. ...Mak, R., Barbie, D., Brennan, P., Hung, R., Amos, C. I. Christiani, D. C., Lin, X. (2020). 
<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/gepi.22358"> Identification of Inflammation and Immune-Related Risk Variants Associated with Squamous Cell Lung Cancer</a>
<i>Genetic Epidemiology</i> 45(1), 99-114



</ul>
</div>


<div id="y2019"><h3>Year 2019</h3>
<ul>

<li>Fagny, M., Platig, J., Kuijjer, M.L., Lin, X., Quackenbush, J. (2019)  
<a href="https://www.nature.com/articles/s41416-019-0614-3/"> Nongenic cancer-risk SNPs affect oncogenes, tumour-suppressor genes, and immune function.</a>
<i>British Journal of Cancer </i> 122, 569-577


<li>Cade, B.E., Chen, H., Stilp, A.M., Louie, T., Ancoli-Israel, S., Arens, R., Barfield, R., Below, J.E., Cai, J., Conomos, M.P., Evans, D.S.,..., Zhu, X., Saxena, R., Lin, X., Redline, S.  (2019)  
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007739/"> Associations of variants In the hexokinase 1 and interleukin 18 receptor regions with oxyhemoglobin saturation during sleep.</a>
<i>PLoS genetics</i> 15(4):e1007739


<li>Liang, J., Cade, B.E., He, K.Y., Wang, H., Lee, J., Sofer, T., Williams, S., Li, R., Chen, H., Gottlieb, D.J. and Evans, D.S., ..., Lin,X., ..., Redline, S., Zhu, X.  (2019)  
<a href="https://www.sciencedirect.com/science/article/pii/S0002929719303908/"> Sequencing Analysis at 8p23 Identifies Multiple Rare Variants in DLC1 Associated with Sleep-Related Oxyhemoglobin Saturation Level.</a>
<i>The American Journal of Human Genetics </i> 105(5), 1057-1068


<li>Chen, S. and Lin, X. (2019)  
<a href="https://pubmed.ncbi.nlm.nih.gov/30590456/"> Analysis in case-control sequencing association studies with different sequencing depths.</a>
<i>Biostatistics </i> 21(3), 577-593


<li>Liu, Z. and Lin, X. (2019)  
<a href="https://amstat.tandfonline.com/doi/abs/10.1080/01621459.2018.1513363#.XiilpEZKhPY"> A Geometric Perspective on the Power of Principal Component Association Tests in Multiple Phenotype Studies. </a>
<i>Journal of the American Statistical Association </i> 114(527), 975-990

<li>Chen, H., Huffman, J., Brody, J. A., Wang, C., Lee, S., Li, Z., Gogarten, S., Sofer, T.,   ..., Redline, S., Smith, N. L.,  Boerwinkle, E., Borecki, I. B., Cupples, L. A., Laurie, C., Morrison, A. C., Rice, K., M., Lin, X. (2019). 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30639324"> Efficient variant set mixed model association tests for continuous and binary traits in large-scale whole genome sequencing studies. </a>
<i>American Journal of Human Genetics</i> 104(2), 260-274


<li>Gaynor, S., Sun, R., Lin, X., and Quackenbush, J. (2019) 
<a href="https://academic.oup.com/bioinformatics/article-abstract/35/22/4568/5486401"> Genetic Variant Set-Based Tests Using the Generalized Berk-Jones Statistic With Application to a Genome-Wide Association Study of Breast Cancer</a>
<i>Bioinformatics</i> 35(22), 4568-4576


<li>Sun, R., Hui, S., Bader, G., Lin, X., and Kraft, P. (2019). 
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007530"> Powerful Gene Set Analysis in GWAS with the Generalized Berk-Jones Statistic. </a>
<i>Plos Genetics</i> 15(3), p.e1007530. 


<li>Liu, Y., Chen, S., Li, Z., Morrison, A. C., Boerwinkle, E., and Lin, X. (2019) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30849328"> ACAT:  Fast and Powerful P-value Combination Method for Rare-variant Analysis in Sequencing Studies. </a>
<i>American Journal of Human Genetics</i> 104(3), 410-421


<li>Li, Z., Li, X., Liu, Y., Shen, J., Chen, H., Zhou, H.,  Morrison, A. C., Boerwinkle, E. and Lin, X. (2019) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30982610"> Dynamic Scan Procedure for Detecting Rare-Variant Association Regions in
Whole-Genome Sequencing Studies </a>
<i>American Journal of Human Genetics</i> 104(5), 802-814


<li>Xu, M., Yao, Y., Chen, H., Zhang, S., Cao, S., Zhang, Z., Luo, B., Liu, Z., Li, Z., Xiang, T., H, G., Feng, Q., Chen, L., Guo, X., Jia, W., Chen, M., Zhang, X., Xie, S., Peng, R., Peng, R., Chang, E, Pedergnana, V., Feng, L., Bei, J., Xu, R., Zeng, M., Ye, W., Adami, H. O., Lin, X., Zhai, W., Zeng, Y. X., Liu, J. (2019). 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31209392"> Genome sequencing analysis identifies Epstein-Barr virus subtypes associated with high risk of nasopharyngeal carcinoma. </a>
<i>Nature Genetics</i> 51(7), 1131-1136


<li>Barfield, R., Wang, H.,  Liu, Y., Brody, J., Swenson, B., Bartz, T. M., Sotoodehnia, N., Chen, Y. I., Cades, B., Chen, H., Patel, S. P., Sunyaev, S. R., Zhu, X., Gharib, S. A., Johnson, W.C., Rotter, J. I., Saxena, R., Purcell, S., Lin, X., Redline, S., Sofer, T. (2019). 
<a href="https://pubmed.ncbi.nlm.nih.gov/31139831"> Epigenome-wide association analysis of daytime sleepiness in the Multi-Ethnic Study of Atherosclerosis reveals African-American specific associations.</a>
<i>Sleep</i> 42(8), p.zsz101.


<li>Peng, C. van Meel, E., Cardenas, A., Rifas-Shiman S., Sonawane, A., Glass, K., Gold, D. R., Platts-Mills, T. A., Lin, X., Oken, E., Hivert, M., Baccarelli, A. A., de Jong, N., Felix, J F., Jaddoe, V., Duijts, L, Litonjua, A. A., DeMeo, D. L, (2019). 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30876376"> Epigenome-wide association study reveals methylation pathways associated with childhood allergic sensitization.</a>
<i>Epigenetics</i> 14(5), 445-466


<li>Wang, H., Cade, B., Sofer, T., Sands, S. A., Chen, H., ...,  Sunyaev, S. R., Cai, J., Lin, X., Purcell, S. M., Laurie, C. C., Saxena, R., Redline, S., Zhu, X. (2019). 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30403821"> Admixture mapping identifies novel loci for obstructive sleep apnea in Hispanic/Latino Americans. </a>
<i>Human Molecular Genetics</i> 28(4), 675-687


</ul>
</div>


<div id="y2018"><h3>Year 2018</h3>
<ul>


<li>Liu, Z. and Lin, X. (2018)  
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28653391"> Multiple Phenotype Association Tests Using Summary Statistics in Genome-Wide Association Studies.</a>
<i>Biometrics </i> 74(1), 165-175
<li>Sun, R., Carroll, R. J., Christiani, D. and Lin, X. (2018) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29120492"> Testing for gene-environment interaction under exposure misspecification. </a>
<i>Biometrics </i> 74(2), 653-663
<li>Gaynor, S., Schwartz, J., and Lin, X. (2018)
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/sim.7945?af=R&"> Mediation analysis for common binary outcomes.</a>
<i>Statistics in Medicine </i> 38(4), 512-529
<li>Huang, J.V., Cardenas, A., Colicino, E., Schooling, C.M., Rifas-Shiman, S.L., Agha, G., Zheng, Y., Hou, L., Just, A.C., Litonjua, A.A. and DeMeo, D.L., Lin, X., Olkin, E., Hivert, M. F., Baccarelli, A.  (2018) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30412002"> DNA methylation in blood as a mediator of the association of mid-childhood body mass index with cardio-metabolic risk score in early adolescence.</a>
<i>Epigenetics</i> 13(10-11), 1072-1087
<li>Carmona, J.J., Barfield, R.T., Panni, T., Nwanaji-Enwerem, J.C., Just, A.C., Hutchinson, J.N., Colicino, E., Karrasch, S., Wahl, S., Kunze, S. and Jafari, N., Lin, X., Schwartz, J., Schulz, H., Barccarelli, A. (2018) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30343628"> Metastable DNA methylation sites associated with longitudinal lung function decline and aging in humans: an epigenome-wide study in the NAS and KORA cohorts.Â </a>
<i>Epigenetics</i> 13(10-11), 1039-1055
<li>Brunst, K.J., Tignor, N., Just, A., Liu, Z., Lin, X., Hacker, M.R., Bosquet Enlow, M., Wright, R.O., Wang, P., Baccarelli, A.A. and Wright, R.J. (2018)
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30343628"> Cumulative lifetime maternal stress and epigenome-wide placental DNA methylation in the PRISM cohort. </a>
<i>Epigenetics</i> 13(6), 665-681

<li>Peng, C., Cardenas, A., Rifas-Shiman, S.L., Hivert, M.F., Gold, D.R., Platts-Mills, T.A., Lin, X., Oken, E., Baccarelli, A.A., Litonjua, A.A. and DeMeo, D.L. (2018) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29692868"> Epigenome-wide association study of total serum immunoglobulin E in children: a life course approach. </a>
<i>Clinical epigenetics</i> 10(1), 55

<li>Chen, S. and Lin, X. (2018) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30590456"> Analysis in case-control sequencing association studies with different sequencing depths. </a>
<i>Biostatistics</i> 


</ul>
</div>



<div id="y2017"><h3>Year 2017</h3>
<ul>
<li>Barnett, I., Murkerjee, R. and Lin, X. (2017) 
<a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2016.1192039"> The Generalized Higher Criticism for Testing SNP-set Effects in Genetic Association Studies. </a>
<i>Journal of the American Statistical Association  </i> 112, 64-76
<li>Sofer, T., Schifano, E., D.,I., Lin, X. (2017) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28346824"> Weighted pseudo-likelihood for SNP set analysis with multiple secondary outcomes in case-control genetic association studies. </a>
<i>Biometrics </i> 73(4), 1210-1220
<li>Barfield, R., Shen, JC, Just, A., Schwartz, J., Baccarelli,  A., VanderWeele, T., and Lin, X. (2017) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29082545"> Testing for the Indirect Effect Under the Null for Genome Wide Mediation Analyses. </a>
<i>Genetic Epidemiology </i> 41(8), 824-833
<li>Chen, J., Behnam, E, Huang, J., Moffatt, M., Schaid, D., Liang, L., Lin, X. (2017)
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3808-1"> Fast and robust adjustment of cell mixtures in epigenome-wide association studies with SmartSVA. </a>
<i>BMC Genomics </i> 18(1), 413



</ul>
</div>



<div id="y2016"><h3>Year 2016</h3>
<ul>
<li>Chen, J. Just, A. C., Schwartz, J., Hou, L, Jafari, L., Sun, Z, Baccarelli, A. and Lin, X. (2016)  <a href="https://www.ncbi.nlm.nih.gov/pubmed/26449931">CpGFilter: Model-based CpG probe filtering with replicates for epigenome-wide association studies.  </a>
<i> Bioinformatics. </i> 32.3, 469-471.
<li> Lin, X., Lee, S.,Wu, M.,Wang, C., Chen H., Li, Z. and Lin, X. (2016) <a href="pub/iSKAT-v20posted.pdf">Test for
rare variants by environment interactions in sequencing association studies.</a>
<i> Biometrics. </i> 72.1, 156-164 <a href="pub/iSKAT-supplementary-v18.pdf"> Supplement. </a> <a href="dat/iSKAT_1.2.tar.gz">Software. </a>
<li>Liang, J., Cade, BE, Wang, H., Chen, H., Larkin, EK,  Saxena, R.,  Lin, X., Redline,S. and  Zhu, X. (2016)    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27027516">Comparison of Heritability Estimation and Linkage Analysis for Multiple Traits Using Principal Component Analyses.  </a>
<i> Genetic Epidemiology. </i> 40.3, 222-232.
<li>Chen, H.,  Wang, C.,  Conomos, MP,  Stilp, AM, Li, Z., Sofer, T.,  Szpiro, AA, Chen, W.,  Brehm, JM, Celedon, JC,  Redline, SS,   Papanicolaou, GP,  Thornton, TA, Laurie, CC,  Rice, K. and Lin, X (2016)  
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27018471">Control for population structure and relatedness for binary traits in genetic association studies using logistic mixed models. </a>
<i>American Journal of Human Genetics.  </i> 98.4, 653-666.
<li>Yung, G. and Lin, X. (2016)  
<a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.21994/abstract">Validity of Using Ad Hoc Methods to Analyze Secondary Traits in Case-Control Association Studies.  </a>
<i>Genetic Epidemiology.  </i> 40.8, 732-743.
<li>Wang, X., Zhang, Z., Morris, N., Cai, T., Lee, S., Wang, C., Timothy, W.Y., Walsh, C.A. and Lin, X. (2016) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27677958">Rare variant association test in family-based sequencing studies.  </a>
<i>Briefings in Bioinformatics.  </i> p.bbw083.  
<li>He, Q., Cai, T., Liu, Y., Zhao, N., Harmon, Q.E., Almli, L.M., Binder, E.B., Engel, S.M., Ressler, K.J., Conneely, K.N., Lin, X.and Wu, M. (2016) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27488097"> Prioritizing individual genetic variants after kernel machine testing using variable selection.   </a>
<i>Genetic Epidemiology.  </i> 40.8, 722-731.
<li>Xing, C., M McCarthy, J., Dupuis, J., Adrienne Cupples, L., B Meigs, J., Lin, X. and S Allen, A. (2016) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27241694"> Robust analysis of secondary phenotypes in case-control genetic association studies.  </a>
<i>Statistics in medicine.  </i> 35.23, 4226-4237.
<li>Liang J, Cade BE, Wang H, Chen H, Gleason KJ, Larkin EK, Saxena R, Lin X, Redline S and Zhu X. (2016) 
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27027516"> Comparison of Heritability Estimation and Linkage Analysis for Multiple Traits Using Principal Component Analyses.  </a>
<i>Genetic Epidemiology.  </i> 40.3, 222-32.







</ul>
</div>
   
<div id="y2015" style="display:none"><h3>Year 2015</h3>
<ul>
<li> Murkerjee, R., Pillai, N.S. and Lin, X. (2015) <a href="http://projecteuclid.org/euclid.aos/1423230083">Hypothesis testing for sparse binary regression. </a>
<i> Annals of Statistics. </i> 43.1, 352-381.
<li> Wang, C., Zhan, X., Liang, L., Abecasis, G. R., and Lin, X. (2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26027497">Improved Ancestry Estimation for both Genotyping and Sequencing Data using Projection Procrustes Analysis and Genotype Imputation. </a>
<i> American Journal of Human Genetics </i> 96, 926-937.
<li> Huang, Y., Liang, L, Moffatt, M., Cookson, W., and Lin, X. (2015).  <a href="https://www.ncbi.nlm.nih.gov/pubmed/25997986">iGWAS: Integrative Genome-Wide Association Studies of Genetic and Genomic Data for Disease Susceptibility Using Mediation Analysis. </a>
<i> Genetic Epidemiology.  </i> 39, 347-359.


</ul>
</div>

<div id="y2014" style="display:none"><h3>Year 2014</h3>
<ul>
<li>Huang. Y.T., VanderWeele, T.J., Lin, X. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3981558/">Joint analysis of SNP and gene expression data in genetic association studies of complex diseases.  </a>
<i>Annals of Applied Statistics.</i> 8.1, 352-376.
<li>Valeri, L., Lin, X., VanderWeele, T. J. (2014) <a href="http://www.ncbi.nlm.nih.gov/pubmed/25220625"> Mediation analysis when a continuous mediator is measured with error and the outcome follows a generalized linear model.</a>
<i>Statistics in Medicine.</i> 33.28, 4875-4890.
<li>Hu, T., Lin, X., Nan, B. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815963/"> Cross-Ratio Estimation for Bivariate Failure Times with Left Truncation. </a>
<i>Life Time Data Analysis. </i> 20.1, 23-37.
<li>Wong, J.Y., De Vivol, I., Lin, X., Grashow, R., Cavallari, J., Christiani, D.C. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4031748/">The association between global DNA methylation and telomere length in a longitudinal study of boilermakers. </a>
<i>Genetic Epidemiology. </i> 38.3, 254-264.
<li>Wong, J.Y.Y., De Vivol, I., Lin, X., Christiani, D.C. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4072226/"> Cumulative PM2.5 exposure and telomere length in Workers exposed to welding fumes. </a>
<i>Journal of toxicology and environmental health. <\i> 77.8, 441-455.
<li>Wong, J.Y.Y., De Vivol, I., Lin, X., Christiani, D.C. (2014) <a href="http://www.ncbi.nlm.nih.gov/pubmed/24475279">The relationship between inflammatory biomarkers and telomere length in an occupational prospective cohort study. </a>
<i>PLoS One.</i> 9.1, e87348.
<li>Barnett, I. and Lin, X. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4350786/">Analytic P-value calculation for the higher criticism test in finite d problems.</a>
<i>Biometrika. </i> 101.4, 964-970.
<li>Sofer, T., Dicker, L., Lin, X (2014) <a href="http://www3.stat.sinica.edu.tw/statistica/J24N4/J24N48/J24N48.html"> Variable selection for high-dimensional multivariate outcomes. </a>
<i>Statistica Sinica. </i> 24.4, 1633-1654.
<li>Wang, C., Zhan, X., Bragg-Gresham, J., Kang, H.M., Stambolian, D., Chew, E.Y., Branham, K., Heckenlively, J., The FUSION Study, Fulton, R., Wilson, R.K., Mardis, E.R., Lin, X., Swaroop, A., Zollner, S., Abecasis, G.R. (2014) Estimating individual ancestry using next generation sequencing.
<i> Nature Genetics. </i> 46.4, 409-415.
<li>Lee, S., Abecasis, G., Boehnke, M., Lin, X. (2014) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4085641/">Rare-variant association analysis: Study designs and statistical tests. </a>
<i>American Journal of Human Genetics. </i> 95.1, 5-23.
</ul>
</div>


   
<div id="y2013" style="display:none"><h3>Year 2013</h3>
<ul>
<li>Ionita-Laza I., Lee, S., Makarov, V., Buxbaum, J.D., Lin, X. (2013) <a href="pub/FB-SKAT_ms.pdf"> Family-based association tests for sequence data, and comparisons with population-based association tests. </a> <i> European Journal of Human Genetics </i>,  21.10, 1158-1162. </li>
<li>Barnett, I. J., Lee, S., and Lin, X. (2013) <a href="pub/CEPSKATmanu+supp.pdf"> Detecting Rare Variant Effects Using Extreme Phenotype Sampling in Sequencing Association Studies</a>. <i>Genetic Epidemiology </i> 37.2, 142-151. </li>
<li> Lin, X., Lee, S., Christiani, D. C., and Lin, X. (2013) <a href="pub/GESAT-main-v24.pdf"> 
Test for the Interaction between
a Genetic Marker Set and Environment in Generalized Linear Models</a>. 
<i>Biostatistics,</i>
doi:10.1093/biostatistics/kxt006. <a href="pub/GESAT-supp-v24.pdf"> Supplemental material</a>. <a href="dat/iSKAT_1.2.tar.gz">Software.</a> </li>
<li>Schifano, E. D., Li, L., Christiani, D. C., and Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3644646/">Genome-wide association
analysis of multiple continuous secondary phenotypes. </a>
<i>American Journal of Human Genetics</i>, 92.5, 744-759.
<li>Wang, X., Lee, S., Zhu, X., Redline, S., and Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4007511/">GEE-Based SNP Set Association Test for Continuous and Discrete Traits in Family-Based Association.</a>
<i>Genetic Epidemiology.</i> 37, 778-786.
<li>Ionita-Laza, I., Lee, S., Makarov, V., Buxbaum, J. D., Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3778346/">Family-based association tests for sequence data, and comparisons with population-based association tests. </a>
<i>European Journal of Human Genetics. </i> 21.10, 1158-1162.
<li>Li, L., Kabesch, M. Bouzigon, E. Demenais, F., Farrall, M. Moffatt, M. F., Lin, X., Liang, L. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3668139/">Using eQTL weights to improve power for genome-wide association studies: A genetic study of childhood asthma.  </a>
<i>Frontiers in Genetics.</i> doi: 10.3389/fgene.2013.00103.
<li>Schifano, E.D., Li, L., Christiani, D.C., Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3644646/"> Genome-wide association analysis of multiple continuous secondary phenotypes. </a>
<i> American Journal of Human Genetics.</i> 92.5, 639-834.
<li>Sofer, T., Baccarelli, A., Cantone, L., Coull, B., Maity, A., Lin, X., Schwartz, J. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3767408/">Exposure to airborne particulate matter is associated with methylation pattern in the asthma pathway. </a>
<i>Epigenomics.</i> 5.2, 147-154.
<li>Lee, S., Teslovich, T.M., Boehnke, M., Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3710762/">General Framework for Meta-analysis of Rare Variants in Sequencing Association Studies. </a>
<i>American Journal of Human Genetics.</i> 93.1, 42-53.
<li>Ionita-Laza, I., Lee, S., Makarov, V., Buxbaum, J.D., Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3675243/">Sequence kernel association tests for the combined effects of rare and common variants.</a>
<i>American Journal of Human Genetics. </i> 92.6, 841-853.
<li>Huang, Y.T. and Lin, X. (2013) <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776447/">Gene set analysis using variance component tests.</a>
<i>BMC Bioinformatics. </i> 14, 210.
</ul>
</div>

<div id="y2012" style="display:none"><h3>Year 2012</h3>
<ul>

<li>Schifano, E., D., Epstein, M. P., Bielak, L. F., Jhun, M. A., Kardia, S. L. R., Peyser, P. A., and Lin, X. (2012) <a href="pub/Schifano_GE_main.pdf">SNP Set Analysis in Genome-wide Association Studies for Familial Data </a>. <i> Genetic Epidemiology </i>, 36, 797-810. <a href="pub/Schifano_GE_supporting.pdf"> Supplemental Materials </a>. </li>
<li>Li, Z., Wang, S. and Lin, X. (2012) <a href="pub/selo-glm-08-06-2012.pdf">Variable selection in Generalized Linear Models using seamless L0 penalty.</a> <i>Canadian Journal of Statistics</i>, 40, 745-769. </li>
<li>Lee, S., Emond, M.J., Bamshad, M.J., Barnes, K.C., Rieder, M.J., Nickerson, D.A., NHLBI GO Exome Sequencing Project-ESP Lung Project Team, Christiani, D.C., Wurfel, M.M. and Lin, X. (2012). <a href="pub/skat-o.pdf">Optimal unified approach for rare-Variant association testing with application to small-sample case-control whole-exome sequencing studies</a>. <i>American Journal of Human Genetics</i>, 91.2, 224-237. <a href="pub/skat-o-suppl.pdf">Supplementary material</a>. </li>
<li>Cai, T., Lin, X. and Carroll, R.J. (2012). <a href="pub/EffScore-V3.pdf">Identifying genetic marker sets associated with phenotypes via an efficient adaptive score test</a>. <i>Biostatistics</i>, 13.4, 776-790. </li>
<li>Dicker, L. and Lin, X. (2012). <a href="pub/DSunique.120412.pdf">Parallelism, uniqueness, and large-sample
asymptotics for the Dantzig selector</a>. <i>The Canadian Journal of Statistics</i>. In press.</li>
<li>Lee, S., Wu, M.C. and Lin, X. (2012). <a href="pub/SKAT-O-Paper_Main.pdf">Optimal tests for rare variant effects in sequencing association studies</a>. <i>Biostatistics</i>, 13.4, 762-775. <a href="pub/SKAT-O-Paper-supplemental.pdf">Supplementary Materials</a>.</li>
<li>VanderWeele, T., Asomaning, K., Christiani, D. C. and Lin, X. (2012). <a href="pub/MediationLC_rev3.pdf">Effect of genetic variants on 15q25.1 on lung cancer for smokers but not through smoking</a>.  <i>American Journal of Epidemiology</i>,175.10, 1013-1020. </li>
<li>Lee, S., Epstein, M.P., Duncan, R. and Lin, X. (2012) <a href="pub/LEDL-10-31-2011_revision2_v2.pdf">Sparse principal component analysis for identifying ancestry-informative markers in genome-wide association studies</a>. <i>Genetic Epidemiology</i>, 36.4, 293-302.</li>
<li>Wang, L., Rotnitzky, A., Lin, X., Millikan, R.E. and Thall, P.F. (2012) <a href="pub/JASA-A10-172R2_FINAL_revision.pdf">Evaluation of viable dynamic treatment regimes in a sequentially randomized trial of advanced prostate cancer</a> (with discussions). <i>Journal of the American Statistical Association</i>, 107.298, 493-508.</li>
<li>Dicker, L., Huang, B. and Lin, X. (2012). <a href="pub/dicker.lin.selo.120718.pdf">Variable selection and estimation with the seamless-L0 penalty</a>. <i>Statistica Sinica</i>. In press. doi: 10.5705/ss.2011.074</li>
<li>Lin, X., Cai, T., Wu, M. C., Zhou, Q., Liu, G., Christiani, D. C. and Lin, X. (2012). <a href="pub/LIN-SKM-GWAS-07052011.pdf">Kernel machine SNP-set analysis for censored survival outcomes in genome-wide association studies</a>. <i>Genetic Epidemiology</i>, 35:620-631. doi: 10.1002/gepi.20610</li>
<li> Yu, Z., Lin, X., and Tu, W. (2012)<a href="pub/splinefrailty_biomREV08.11.pdf"> Additive frailty models for correlated survival data with application to sexually transmitted infection data</a>. <i> Biometrics</i>, 68.2, 429-436.</li>
</ul>
</div>

<div id="y2011" style="display:none"><h3>Year 2011</h3>
<ul>
<li>Ionita-Laza, I., Makarov, V.,  Yoon, S.,  Raby, B.,  Buxbaum, J.,   Nicolae, D. L.,  Lin, X. (2011) <a href="pub/Mendelian_september16_1.pdf">Finding genes in Mendelian disorders using sequence data: methods and applications</a>. <i>American Journal of Human Genetics</i>, Vol 89, Issue 6, 701-712</li>
<li> Hu,  T.,  Nan, B.,  Lin, X., and  Robins, J. (2011). Time Dependent Cross-Ratio Estimation. <i>Biometrika</i>, 98, 341-354.</li>
<li>Huang, Y. T., Lin, X., Liu, Y., Chirieac, L. R., McGovern, R., Wain, J., Heist, R., Skaug, V., Zienolddiny, S., Haugen, A., Su, L., Fox, E. A., Wong, K. K. and Christiani, D. C. (2011) <a href="pub/Huang-PNAS-2011.pdf">Cigarette smoking increases copy number alterations in nonsmall-cell lung cancer</a>. <i>Proceedings of the National Academy of Sciences</i>, 108:16345-16350. doi: 10.1073/pnas.1102769108. <a href="pub/Huang-PNAS-2011-Appendix.pdf">Appendix</a>.</li>
<li> Wu, M., Lee, S., Cai, T., Li, Y., Boehnke, M., and Lin, X. (2011) <a href="pub/SKAT-paper-and-supplement.pdf">Rare Variant Association Testing for Sequencing Data Using the Sequence Kernel Association Test (SKAT)</a> <i> American Journal of Human Genetics,</i> 89, 82-93. </li>
<li> Lee,S., Wu, M., Cai, T., Li, Y., Boehnke, M., and Lin, X. (2011) <a href="pub/SampleSize_Technical_Report.pdf"> Power and sample size calculations for designing rare variant sequencing association studies.</a> <i> Manuscript,</i></li>
<li> Schwartzman, A. and Lin, X. (2011) <a href="pub/corrFDR_Biometrika2.pdf"> The effect of correlation on false discovery rate estimation.</a> <i> Biometrika,</i> 98, 199-214.</li>
<li> Cai, T., Tonini, G., Lin, X. (2011) <a href="pub/SurvivalScoreTest-V3.pdf"> Kernel machine approach to testing the significance of multiple genetic markers for risk prediction.</a> <i> Biometrics, </i> 67, 975-986.</li>
<li> Maity, A. and Lin, X. (2011) <a href="pub/Rev03_kernel_interaction_test.pdf">Powerful tests for detecting a gene effect in the presence of possible gene-gene interactions using garrote kernel machines. </a> <i> Biometrics</i>, 67, 1271-1284.</li>
</ul>
</div>

<div id="y2010" style="display:none"><h3>Year 2010</h3>
<ul>
<li> Wu, M. C., Kraft, P., Epstein, M. P.,Taylor, D., M., Chanock, S. J., Hunter, D., J., and Lin, X. (2010) <a href="pub/gwas-kmtest.pdf"> Powerful SNP Set Analysis for Case-Control GenomeWide Association Studies.</a> <i> American Journal of Human Genetics, </i>, 86, 929-942.</li>
<li>Wang, L., Rotnitzky, A., Lin, X. (2010) <a href="pub/indep-nonpar-missing-data-merged.pdf">Nonparametric regression with missing outcomes using weighted kernel estimating equations.</a> <i> Journal of the American Statistical Association,</i>  105, 1135-1146.</li>
<li> Dicker, L., Lin, X., Ivanov, A. (2010) <a href="pub/proteomic.MCP.070110.pdf"> Increased Power for the Analysis of Label-Free LC-MS/MS Proteomic Data by Combining Spectral Counts and Peptide Peak Attributes. </a>  <i> Molecular and Cellular Proteomics, </i>, doi: 10.1074/mcp.M110.002774. <a href="pub/supplemental.MCP.051810"> Supplemental Material. </a> </li>
<li> Long, Q., Little, R. and Lin, X. (2010) Estimating Causal Effects in Trials Involving Multi-Treatment Arms Subject to Non-compliance: A Bayesian Framework. <i> Journal of the Royal Statistical Society, Series C</i>,  59(3), 513-531.</li>
<li>Ye, W., Taylor, J. M., Lin, X.  (2010). Simulating Pseudo Data for a Two-stage Approach in Joint Modeling of Longitudinal Measurements and Time-to-Event Data. <i> Biometrics,</i> 66, 987-991.</li>
</ul>
</div>

<div id="y2009" style="display:none"><h3>Year 2009</h3>
<ul>
<li> Wu, M.,C., Zhang, L., Wang, Z., Christiani, D. C.,  Lin, <a href="pub/slda-bioinformatics.pdf"> Sparse linear discriminant analysis for simultaneous gene set/pathway significance test and gene selection. </a>, <i> Bioinformatics, </i>, 25,1145-1151.</li>
<li> Pan, W., Zeng, D. and Lin, X. (2009) <a href="pub/transition2.pdf">Semiparametric transition  measurement error models for longitudinal data. </a>, <i> Biometrics,</i>, in press. </li>
<li> Little, R., Long, Q. and Lin, X. (2009) A comparison of methods for estimating the causal effect of a treatment in randomized clinical trials subject to noncompliance. <i> Biometrics,</i> 65,  640-649. </li>
<li> Li, Y., Tang, H. and Lin, X. (2009). Covariate measurement error in spatial linear mixed models. <i> Statistica Sinica,</i> 19, 1077-1093.</li>
<li> Li, Y., Lin, X. and Muller, P. (2009) Bayesian inference in semiparametric mixed models for longitudinal data. <i> Biometrics</i>,  in press.</li>
</ul>
</div>

<div id="y2008" style="display:none"><h3> Year 2008 </h3>
<ul>
<li> Liu, D., Ghosh, D. and  Lin, X. (2008)  <a href="pub/logistic_km.pdf"> Estimation and Testing for the Effect of a Genetic Pathway on a Disease Outcome Using Logistic Kernel Machine Regression via Logistic Mixed Models. </a><i> BMC Bioinformatics,</i> 9, 292. <a href="pub/logistic_kernel_machine_semiparametric_regression.sas">SAS code </a></li>
<li><a> Kwee, L, Liu, D., Lin, X., Ghosh, D., and Epstein, M. (2008) </a><a href="pub/lskm_qtl.pdf"> A powerful and flexible multilocus association test for quantitative traits. </a> <i> American Journal of Human Genetics,</i> 82, 386-397.</li>
<li> Yu, Z. and Lin, X. (2008) <a href="pub/survival_nonpar.pdf"> Nonparametric regression using local kernel estimating equations for correlated failure time data.</a> <i> Biometrika, </i>, 95, 123-127.</li>
<li> Long, Q., Little, R. J., Lin, X.  (2007) <a href="pub/hybrid.pdf"> Causal inference in hybrid intervention trials involving treatment choice.</a> <i> Journal of the American Statistical Association,</i> 103, 474-484</li>
<li> Little, R., Long, Q. and Lin, X. (2008) <a href="pub/cacemem.pdf"> A comparison of methods for estimating the causal effect of a treatment in randomized clinical trials subject to noncompliance, </a> <i> Biometrics, </i> in press.</li>
<li> Zeng, D., Lin, D., and Lin, X. (2008) Semiparametric linear transformation models with random effects for clustered time data. <i> Statistica Sinica</i>, 18, 355-377.</li>
<li> Li, Y., Prentice, R. L. and Lin, X. (2008) <a href="pub/li08.pdf"> Maximum likelihood estimation in semiparametric normal transformation models for bivariate survival data.</a> <i> Biometrika</i>,95, 947-960.</li>
<li>Harezlak, J., Wu, M. C., Wang, M., Schwartzman, A., Christiani, D. C., and Lin, X. (2008) <a href="pub/bangaldesh.pdf"> Biomarker discovery for arsenic exposure using functional data analysis and feature learning of mass spectrometry proteomic data.</a><i> Journal of Proteome Research </i>, 7, 217-224</li>
<li>Schipper, M., Taylor, J. and Lin, X. (2008) Generalized monotonic functional mixed models with application to modeling normal tissue complications. <i> Journal of the Royal Statistical Society, Series C</i>, 57,149-163.</li>
</ul>
</div>
 
<div id="y2007" style="display:none"><h3>Year 2007</h3>
<ul>
<li>Liu, D., Lin, X. and Ghosh, D. (2007) <a href="pub/lskm.pdf"> Semiparametric Regression of Multi-Dimensional Genetic Pathway Data: Least Squares Kernel Machines and Linear Mixed Models.</a> <i> Biometrics</i>, 63, 1079-1088.<b> This paper won the 2007 Best Biometrics Paper Award.</b><a href="pub/kernel_machine_semiparametric_regression.sas"> SAS code </a></li>
<li>Schipper, M., Taylor, J. and Lin, X. (2008) Bayesian generalized monotonic functional mixed models for the effects of radiation dose histograms on normal tissue complications. <i> Statistics in Medicine</i>,  26, 4643-4656.</li>
<li> Ye, W., Lin, X. and Taylor, J. (2007) <a href="pub/yebiometrics.pdf"> Semiparametric modeling of longitudinal measurements and time-to-event data--A two-stage regression calibration approach.</a> <i> Biometrics</i>, in press.</li>
<li> Harezlak, J., Wang, M., Christiani, D. and Lin, X. (2007) <a href="pub/BinfmcsQuantFracJun07.pdf"> Quantitative quality-assessment techniques to compare frantionation and depletion methods in SELDI-TOF mass spetrometry experiments.</a><i> Bioinformatics </i>, 23, 2441-2448.</li>
<li> Zhang, D., Lin, X. and Sowers, M. F.  (2007)<a href="pub/func_mixed.pdf"> Two-Stage Functional Mixed Models for Evaluating the Effect of Longitudinal Covariate Profiles on a Scalar Outcome. </a> <i> Biometrics,</i> 63, 351-362.</li>
<li> Pan, W., Lin, X. and Zeng, D. (2007). <a href="pub/pan05.pdf"> Structural Inference in Transition Measurement Error Models for Longitudinal Data.</a> <i> Biometrics </i>, in press.</li>
</ul>
</div>

<div id="y2006" style="display:none"><h3>Year 2006</h3>
<ul>
<li> Lin, X. and Carroll, R. J. (2006) <a href="pub/lin_carroll_repeatedmeasures01.pdf"> Semiparametric Estimation in General Repeated Measures Problems.</a><i> Journal of the Royal Statistical Society, Series B,</i> 68, 69-88.</li>
<li> Li, Y. and Lin, X. (2006) <a href="pub/li_lin05.pdf"> Semiparametric normal transformation models for spatially correlated survival data.</a> <i> Journal of the American Statistical Association.</i> 101, 591-603.</li>
<li> Nan, B., Lin, X., Lisabeth, L. D., and Harlow, S. D. (2006) <a href="pub/crossratio_v4.pdf"> Piece-wise constant cross-ratio estimates for the assocation between age at a marker event and age at menopaus.</a> <i> Journal of the American Statistical Association.</i> 101, 65-77.</li>
</ul>
</div>

<div id="y2005" style="display:none"><h3>Year 2005</h3>
<ul>
<li> Roy, J. and Lin, X. (2005). <a href="pub/bmtcs_021201R2_v1.pdf"> Missing Covariates in Longitudinal Data With Informative Dropouts: Bias Analysis and Inference.</a> <i>Biometrics</i>, 61, 837-846.</li>
<li> Hogan, J. W., Lin, X., Herman, B. (2005) <a href="pub/vcm03.pdf"> Mixtures of Varying Coefficient Models for Longitudinal Data Discrete or Continuous Non-ignorable Dropout. </a><i>Biometrics</i>, 60, 854-864.</li>
<li> Wang, N., Carroll, R. J. and Lin, X. (2005). <a href="pub/semi03.pdf"> Efficient semiparametric marginal estimation for longitudinal/clustered data. </a> <i> Journal of the American Statistical Association</i>, 100, 147-157.</li>
<li> Nan, B., Lin, X., Lisabeth, L. D., Harlow, S. (2005). <a href="pub/marker_v6.pdf"> A varying-coefficient Cox model for the effect of age at a marker event on age at menopause. </a> <i>Biometrics</i>, 61, 576-583.</li>
<li> Bellamy, S. L., Li, Y. Lin, X., and Ryan, L. M. (2005) <a href="pub/bellamy.pdf"> Note on the PQL bias in estimating cluster-level covariate effects in group-randomized trials. </a> <i> Statistica Sinica</i>, 15, 1015-1032.</li>
</ul>
</div>

<div id="y2004" style="display:none"><h3>Year 2004</h3>
<ul>
<li> Carroll, R. J., Hall, P., Apanasovich, T. V., and Lin, X. (2004)  <a href="pub/linked14.pdf"> Histospline method in nonparametric regression models with application to longitudinal/clustered data.</a><i> Statistical Sinica</i>, 14, 649-674.</li>
<li> Lin, X., Wang, N., Welsh, A. and Carroll, R. J. (2004) <a href="pub/spline.pdf"> Equivalent Kernels of Smoothing Splines in Nonparametric Regression for Clustered Data.</a> <i>Biometrika</i>,  91, 177-193.</li>
<li> Li, L, Lin X., Brown, M. B., Gupta S., Lee, K. H. (2004) <a href="pub/MEPKv6.pdf"> A population pharmacokinetic model with time-dependent covariates measured with error. </a> <i> Biometrics</i>, 60, 451-460.</li>
<li> Linton O. B., Mammen E., Lin X., Carroll, R. J. (2004). Accounting for correlation in marginal longitudinal nonparametric regression. <i> Proceedings of the Second Seattle Symposium in Biostatistics</i>, editor, D. Lin and P. Heagerty (Editors), 23-34.</li>
</ul>
</div>

<div id="y2003" style="display:none"><h3>Year 2003</h3>
<ul>
<li> Li, Y. and Lin, X. (2003). Functional inference in frailty measurement error models using the SIMEX approach. <i> Journal of the American Statistical Association</i>,  98, 191-203.</li>
<li> Zhang, D. and Lin, X. (2003). Hypothesis testing in semiparametric additive mixed models. <i>Biostatistics</i>, 4, 57-74.</li>
<li> Li, Y. and  Lin, X. (2003). Testing the correlation for clustered categorical and  censored discrete failure time data when covariates are measured without/with errors. <i> Biometrics</i>, 59, 25-35.</li>
<li>Roy, J., Lin, X., and Ryan, L. (2003). Scaled Marginal Models For Multiple Continuous Outcomes. <i> Biostatistics </i>, 4, 371-384.</li>
<li> Epstein, M.  Lin, X. and M. Boehnke (2003). A tobit variance-component method for linkage analysis of truncated trait data. <i> American Journal of Human Genetics</i>, 72, 611-620.</li>
</ul>
</div>

<div id="y2002" style="display:none"><h3>Year 2002</h3>
<ul>
<li> Roy, J. and Lin, X. (2002) Analysis of multivariate longitudinal outcomes with non-Ignorable dropouts and missing covariates: changes in methadone treatment practices. <i> Journal of the American Statistical Association</i>, 97, 40-52.</li>
<li> Welsh, A. H.,   Lin, X. and Carroll, R. J. (2002). Marginal longitudinal nonparametric regression: Locality and efficiency of spline and kernel methods. <i> Journal of the American Statistical Association, </i>, 97, 482-493.</li>
<li> Epstein, M. Lin, X. and M. Boehnke (2002). Ascertainment-adjusted parameter estimates revisted. <i> American Journal of Human Genetics </i>, 70, 886-895.</li>
</ul>
</div>

<div id="y2001" style="display:none"><h3>Before Year 2002</h3>
<ol>
<li> Breslow N. E. and Lin X. (1995). Bias correction in generalized linear mixed models with a single component of dispersion. <i> Biometrika </i>, <b> 82, </b> 81-91.</li>
<li> Lin X. and Breslow N. E. (1996). Bias correction in generalized linear mixed models with multiple components of dispersion. <cite><i>Journal of the American Statistical Association</i></cite>, <b> 91, </b> 1007-1016.</li>
<li>Lin X and Breslow NE (1996). Analysis of correlated binomial data in logistic-normal models. <cite> <i> Journal of Statistical Computation and Simulation</i></cite>, 55, 133-146.</li>
<li> Lin X. (1997). Variance component testing in generalized linear models with random effects, <cite> <i>Biometrika</i></cite>, 84, 309-326. </li>
<li> Lin X., Raz J., and Harlow S. (1997). Linear mixed models with heterogeneous within-cluster variances,  <cite> <i>Biometrics</i> </cite>, 53, 910-923.</li>
<li>Lin X. and Little R. J. A. (1997). Disucssion of ``Analysis of nonrandomly censored ordered categorical longitudinal data from analgesic trials,'' by Sheiner L, Beal S. and Dunne A. <cite> <i>Journal of the American Statistical Association</i></cite>, 92, 1245-1247.</li>
<li>Carroll R. J., Lin X. and Wang, N. (1997).  Generalized Linear Mixed Measurement Error Models. In Modeling Longitudinal and Spatially Correlated Data, T. Gregoire, editor. Springer Verlag, New York. </li>
<li>Wolfinger R. D., and Lin X. (1997). Two Taylor-series approximations methods for nonlinear mixed models. <cite><i> Journal of Computational Statistics and Data Analysis</i></cite>, 25, 465-490.</li>
<li>Wang N., Lin X., Gutierrez R., and Carroll R. J. (1998). Bias analysis and SIMEX inference in generalized linear mixed measurement error models , <cite> <i>Journal of the American Statistical Association</i></cite>, 93, 249-261.</li>
<li>Zhang D., Lin X., Raz J., and Sowers M. (1998). Semiparametric stochastic mixed models for longitudinal data, <cite> <i>Journal of the American Statistical Association</i></cite>, 93, 710-719.</li>
<li>Lin X. and Carroll R. J. (1999). SIMEX variance component tests in generalized linear mixed measurement error models. <cite> <i> Biometrics</i></cite>, 55, 613-619.</li>
<li>Lin X. and Zhang D. (1999). Inference in generalized additive mixed models using smoothing splines, <cite> <i> Journal of the Royal Statistical Society, Series B</i></cite>, 61, 381-400.</li>
<li>Sammel M., Lin X. and Ryan L. (1999). Multivariate linear mixed models for multiple outcomes, <cite> <i>Statistics in Medicine,</i></cite> 18, 2479-2492.</li>
<li>Wang, N. Lin, X., and  Gutierrez, R. (2000). A bias correction regression calibration approach in generalized linear mixed measurement error model.  <cite><i> Communication in Statistics, Series A, Theory and Methods,</i></cite>, 28, 217--232.</li>
<li>Zhang D., Lin X. and Sowers M. (2000). Periodic semiparametric regression for longitudinal hormone data from multiple menstrual cycles. <cite><i> Biometrics, </i></cite>, 56, 31-39. </li>
<li>Lin, X., Ryan, L, Sammel, M., Zhang, D., Padungtod, C., Xu, X. (2000). A scaled linear mixed model for multiple continuous outcomes. <cite> <i>Biometrics,</i> </cite> 56, 593-601.</li>
<li>Lin, X. and Carroll, R. J. (2000). Nonparametric function estimation for clustered data when the predictor is measured without/with error. <cite> <i> Journal of the American Statistical Association</i> </cite>, 95,520-534.</li>
<li>Roy, J. and Lin, X. (2000). Latent variable models for longitudinal data with multiple continuous outcomes. <cite> <i>Biometrics,</i> </cite> 56,1047-1054.</li>
<li>Li, Y. and Lin, X. (2000). Covariate measurement errors in frailty models for clustered survival data. <i>Biometrika</i>,  87, 849-866.</li>
<li> Lin, X. and Carroll, R. J. (2001). Discussion of ``Semiparametric and nonparametric analysis of longitudinal data,'' by D. Y. Lin and Z. Ying. <i> Journal of the American Statistical Association </i>, 96, 114-115.</li>
<li> Lin, X. and Carroll, R. J. (2001). Semiparametric regression for clustered data using generalized estimating equations, <i> Journal of the American Statistical Association</i>,  96, 1045-1056.</li>
<li>Lin, X.  and Carroll, R. J. (2001). Semiparametric regression for clustered data. <i> Biometrika</i>, 88, 1179-1865.</li>
<li>Lin, X. and Zhang, D. (2001).  Discussion of ``Semiparametric nonlinear mixed effects mdoels and their applications" by C. Ke and Y. Wang. <i> Journal of the American Statistical Association, </i>  96, 1288-1291.</li>
</ol>
</div>

<div class="back2top"><a href="#top">Back to top</a></div>

</div>


<div class="footnote">
Copyright &copy; Xihong Lin, 2010-2023
</div>

</div>

</body>
</html>
